catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/util/EndpointUtil.java

```diff
@@ -392,6 +409,55 @@ public String getJson(Object result) {
     return objectMapper.toJson(result);
   }
 
+  public CqlQueryResponse executeCqlQuery(CqlRequest cqlRequest)
```

Comment:
❓ Just curious what motivations for the move to EndpointUtil

Reply:
it is being used in both QueryApplication and CqlTransformHandler so rather than duplicate the method it was moved to a utility class